arviz/data/inference_data.py

```diff
@@ -337,7 +338,9 @@ def items(self) -> "InferenceData.InferenceDataItemsView":
 
     @staticmethod
     def from_netcdf(
-        filename, *, engine="h5netcdf", group_kwargs=None, regex=False
+        filename, *,
+        engine="h5netcdf", group_kwargs=None, regex=False,
+        base_group: Optional[str] = None,
```

Comment:
```suggestion
        base_group: str = "/",
```

I think using this as default will be easier to follow and will simplify the code.